src/core/views.py

```diff
@@ -244,7 +244,11 @@ def user_login_orcid(request):
                 if candidates.exists():
                     # Store ORCID for future authentication requests
                     candidates.update(orcid=orcid_id)
-                    login(request, candidates.first())
+                    login(
+                        request,
+                        candidates.first(),
+                        backend='django.contrib.auth.backends.ModelBackend',
+                    )
```

Comment:
This is slightly different from the flow we do right above: https://github.com/openlibhums/janeway/blob/f014424815b9ea06990e983d15e1879c9cf5a687/src/core/views.py#L233

I think I prefer this approach to mutation. Can we change the above to use the same `kwarg` in `login()`, for consistency'sake?